testing/environments/snapshot.yml

```diff
@@ -21,4 +25,6 @@ services:
     build:
       context: ./docker/kibana
       dockerfile: Dockerfile-snapshot
-
+      args: [ ]
+        #KIBANA_DOWNLOAD_URL: https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha1-SNAPSHOT-linux-x86_64.tar.gz
```

Comment:
I'm hoping to find a solution to be able to uncomment it and use the "pure" docker file from the infra team. But I didn't want to spend too much time right now on figuring out on how this is possible with version 2 of the docker compose file.